cms/djangoapps/contentstore/tests/test_exams.py

```diff
@@ -55,6 +56,7 @@ def setUp(self):
     def _get_exams_url(self, course_id):
         return f'{settings.EXAMS_SERVICE_URL}/exams/course_id/{course_id}/'
 
+    @skip("OldMongo Deprecation")
     @ddt.data(
```

Comment:
ok so I found what's going on and put up a PR to fix these tests specifically: https://github.com/openedx/edx-platform/pull/31223

Tests related to proctoring scaffold out test data using the field `is_proctored_exam` which is an alias to the underlying `is_proctored_enabled` value. This alias appears to not work when setting up data in split modulestore. It's not entirely obvious to me if this intended, a bug, or this one field is a really gross edge case. The fastest fix is to just use the underlying field name instead so I'm just doing that in my linked PR.

I've also left a comment here because this hiding some tests that should be failing